### ISSUE TITLE
on the latest release 0.18.0  and npm package version 0.18.0 , lack of some functions

### ISSUE THREAD

**Issue by jjeejj** [axios/axios#1519]
on the latest release 0.18.0  and npm package version 0.18.0 ,the #869  PR feature not merged
eg: responseEncoding config

**Comment by jjeejj** [comment]
you can download 0.18.0 version on releases tab or exec npm install asiox 
see the Source code to check it

**Comment by JaakkoL** [comment]
I came across this same thing when trying to figure out why [Googleapis library](https://www.npmjs.com/package/googleapis) is unable to fix character encoding when fetching files from Google Drive. It uses Axios for making http requests and it assumes that the response encoding can be passed as a [config parameter](https://www.npmjs.com/package/googleapis#request-level-options).

After doing some digging, I realised that release 0.18.0 has fixed encoding to `utf-8` while https://github.com/axios/axios/pull/869 PR has already solved this issue.

Any chance on getting a release with the aforementioned PR included?
